.circleci/config.yml

```diff
@@ -34,6 +34,30 @@ jobs:
       - run:
           when: on_fail
           command: etc/testing/circle/kube_debug.sh
+  test-publish:
+    docker:
+      - image: circleci/python:3.7
+    steps:
+      - checkout
+      - run:
+          name: Init .pypirc
+          command: |
+            echo -e "[testpypi]" >> ~/.pypirc
+            echo -e "username = $PYPI_USERNAME" >> ~/.pypirc
```

Comment:
```suggestion
            echo -e "username = $TEST_PYPI_USERNAME" >> ~/.pypirc
```

Reply:
Although the username would most likely always be `__token__`. I'm assuming eventually we would have two sets of env variables in Circle. `TEST_PYPI_PASSWORD` and `PYPI_PASSWORD`

Reply:
yes, these are set

Reply:
I see, my assumption was that env variables are set globally for a project. So if we have two jobs one for testpypi and another for real pypi, then wouldn't we need both `TEST_PYPI_PASSWORD` and `PYPI_PASSWORD`?